packages/teleterm/src/main.ts

```diff
@@ -55,6 +59,16 @@ app.whenReady().then(() => {
     });
   }
 
+  const isWin = mainProcess.settings.platform === 'win32';
+
+  const installPath = app.getAppPath();
+  interceptFileProtocol({ protocol, isWin, installPath, logger });
```

Comment:
Hmm would it be possible to export only one function from the module, like `enableWebHandlersProtection()`? Then I would rename `installWebHandler` to `blockUnusedProtocols`. 
From looking at the functions calls it's hard to tell why we need them and what they do 🤔

Moreover, you don't have to pass `protocol` and `logger` to the function. You can just import `protocol`, because you're in the scope of the main process. 
`logger` can be created in the file directly (`const logger = new Logger('WebHandlerProtection')`), but it's up to you.

Reply:
exporting just one function and moving to the default node url method removed about half the code. THANK YOU!!

Reply:
💪
I was a bit imprecise when it comes to naming (I used single form in `new Logger('WebHandlerProtection')`) , let's name the function enableWebHandler**s**Protection, because we are handling multiple protocols. 
And also, I think, I liked that separation into two functions, it's easier to read, so I would see it this way:

```typescript
export function enableWebHandlersProtection({
  installPath,
}: {
  installPath: string;
}) {
  interceptFileProtocol();
  blockUnusedProtocols();
}
``` 
But this is a matter of preference, so I leave it to you.